numpy/doc/basics.py

```diff
@@ -314,8 +314,8 @@
 ``np.clongdouble`` for the complex numbers). You can find out what your
 numpy provides with ``np.finfo(np.longdouble)``.
 
-NumPy does not provide a dtype with more precision than C
-``long double``\\s; in particular, the 128-bit IEEE quad precision
+NumPy does not provide a dtype with more precision than C's
+``long double``\\; in particular, the 128-bit IEEE quad precision
 data type (FORTRAN's ``REAL*16``\\) is not available.
```

Comment:
This second one is still going to cause trouble I think. Does replacing `\\` with `\\ ` solve the problem?

Reply:
Maybe. I'm not sure to understand what the double backslash does. Does it print one backslash? Do we want that? Are they needed?

Reply:
The first backslash is a python escape character. The second is an rst escape character, which seems to be a failed attempt to solve the problem you identifies. If you add a space after the double backslash, then rst will se `\ `, which I believe means "don't emit a space, but start a new parser token"